theseus/labs/lie/lie_tensor.py

```diff
@@ -388,28 +427,39 @@ def as_lietensor(
     data: Any,
     ltype: Optional[_ltype] = None,
     dtype: torch.dtype = torch.float,
-    device: Union[torch.device, str] = None,
+    device: DeviceType = None,
 ) -> _LieTensorBase:
     if isinstance(data, LieTensor):
-        return data
+        data.to()
```

Comment:
If passed data is on GPU does calling default `.to()` move it to CPU and then back to GPU below in line 436?

Reply:
Oops, this shouldn't be here, I wrote it so I could click into the function code with the IDE lol. Sorry about that.

Reply:
No worries, thanks for updating.